src/sockets.js

```diff
@@ -92,18 +194,18 @@ module.exports = (
       logger.info('New socket is NOT one time use')
       // this is where we create the websocket connection
       // with the GunDB service.
-      Mediator.createMediator(socket);
+      Mediator.createMediator(socket)
```

Comment:
Looks like an LND socket would also get that mediator attached to it?

Reply:
Yeah, I'll need to get rid of that on LND sockets so we don't confuse fails related to the Mediator socket with ones that are related to the LND one.